### PR TITLE
Resource model for ios lag interfaces

### PR DESCRIPTION
**Pull request by justjais**
Signed-off-by: Sumit Jaiswal <sjaiswal@redhat.com>

- ios_lag_interfaces resource model

### CODE REVIEW

**Comment by justjais** [comment]
recheck

**Comment by justjais** [comment]
recheck

**Comment by ikhan2010** [comment]
@NilashishC please review when you get a moment.

File: models/ios/lag_interfaces/override_example_01.txt [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+#  channel-group 30 mode active
+
+- name: Override device configuration of all interfaces with provided configuration
+  ios_interfaces:
+    config:
+      - name: 20
+        mode: auto
+        members:
+        - GigabitEthernet0/2
+        - GigabitEthernet0/2
```
Comment (ikhan2010) [inline models/ios/lag_interfaces/override_example_01.txt]: @justjais something is not right with this example.  Do you intend to use GigabitEthernet0/2 in this example twice?  I think it should be GigabitEthernet0/3

**Comment by ikhan2010** [comment]
@justjais looks good.  I think you have a typo in the examples.  please check that.

**Comment by justjais** [comment]
@ikhan2010 Thanks for the review, and I have updated the override example type. :)

File: models/ios/lag_interfaces/override_example_01.txt [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+#  channel-group 10 mode auto
+# interface GigabitEthernet0/3
+#  shutdown
+#  channel-group 20 mode on
+# interface GigabitEthernet0/4
+#  shutdown
+#  channel-group 30 mode active
+
+- name: Override device configuration of all interfaces with provided configuration
+  ios_interfaces:
```
Comment (samccann) [inline models/ios/lag_interfaces/override_example_01.txt]:
```suggestion
  ios_lag_interfaces:
```

File: models/ios/lag_interfaces/replaced_example_01.txt [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+#  channel-group 10 mode auto
+# interface GigabitEthernet0/3
+#  shutdown
+#  channel-group 20 mode on
+# interface GigabitEthernet0/4
+#  shutdown
+#  channel-group 30 mode active
+
+- name: Replaces device configuration of listed interfaces with provided configuration
+  ios_interfaces:
```
Comment (samccann) [inline models/ios/lag_interfaces/replaced_example_01.txt]:
```suggestion
  ios_lag_interfaces:
```

**Pull request merged**